src/securejoin.rs

```diff
@@ -626,16 +626,17 @@ pub(crate) fn handle_securejoin_handshake(
             let ret = if join_vg {
```

Comment:
can we rename `ret` into `abort_retval` or something else now?  I think that'd be clearer

Reply:
yip, this is a good idea, i do a thing.